### PR TITLE
Footer - Template

### LINKED ISSUE #213: Footer for the page || GSSOC'22
I would like to take up the task of improving the footer of the site and make a beautiful, interactive and responsive footer.

I am also attaching a footer that I have made on one of my own sites to serve for credibility.

Screenshot 2022-02-27 201107

Please give suggestions about the type of footer you'd want. 

Could you assign this to me @arpit456jain ? And I will try to improve the look and feel of the footer section and submit a PR as soon as possible.
Thank You !!

### PR DESCRIPTION
**Pull request by Rishabh-Gaud**
# Description

I am add Footer Template for website

Fixes #213

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Explain the Testing instructions

**Test Configuration**:

- Text-editors used:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
image

### CODE REVIEW

**Review by VidhiBhatt01 — COMMENTED** [review]
This one is good but PA requested something like this ig

image

**Comment by Rishabh-Gaud** [comment]
> This one is good but PA requested something like this ig
> 
> image

Okay ma'am @VidhiBhatt01 , I will made changes asap

**Comment by VidhiBhatt01** [comment]
@arpit456jain

File: Footer-Template/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            <ul>
+                <li>Jaunpur</li>
+                <li>UP </li>
+                <li>+91-9876543210</li>
+                <li>rishabh.kumbh@gmail.com</li>
+            </ul>
+        </div>
+    </div>
+
+    <p class="footer__copy">&#169; 2021 rimmi_goWithFlow. All right reserved</p>
```
Comment (VidhiBhatt01) [inline Footer-Template/index.html:59]: ??

**Pull request merged**